adminSiteServer/publicApiRouter.ts

```diff
@@ -18,3 +22,12 @@ publicApiRouter.router.get(
         }
     }
 )
+
+publicApiRouter.router.get("/health", async (req: Request, res: Response) => {
+    const sqlPromise = db.mysqlFirst(`SELECT id FROM charts LIMIT 1`)
+    const timeoutPromise = rejectAfterDelay(1500) // Wait 1.5 seconds at most
+    const result = await Promise.race([sqlPromise, timeoutPromise])
+    if (result instanceof Error)
```

Comment:
This check currently doesn't work, because `await` will throw and you'll have to use a try-catch block.